treeherder/model/sql/template_schema/project.sql.tmpl

```diff
@@ -377,12 +381,16 @@ DROP TABLE IF EXISTS `revision`;
 CREATE TABLE `revision` (
   `id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,
   `revision` varchar(50) COLLATE utf8_bin NOT NULL,
+  `short_revision` varchar(12) COLLATE utf8_bin DEFAULT NULL,
+  `long_revision` varchar(40) COLLATE utf8_bin DEFAULT NULL,
   `author` varchar(150) COLLATE utf8_bin NOT NULL,
   `comments` text DEFAULT NULL,
   `repository_id` int(11) unsigned NOT NULL,
   `active_status` enum('active','onhold','deleted') COLLATE utf8_bin DEFAULT 'active',
   PRIMARY KEY (`id`),
   UNIQUE KEY `idx_revision` (`revision`, `repository_id`),
+  UNIQUE KEY `idx_short_revision` (`short_revision`, `repository_id`),
```

Comment:
The ``short_revision`` will be the same value as the ``revision`` field is, currently.  So it would have the same collision rate we currently have (which I don't believe hits any collisions).  But I agree that these should match the ``result_set`` table and don't need to be unique.  So...  I'll do that.  :)